ui/js/controllers/perf/compare.js

```diff
@@ -1,7 +1,6 @@
 // Remove the eslint-disable when rewriting this file during the React conversion.
 /* eslint-disable func-names, object-shorthand, prefer-destructuring, prefer-template, radix */
 import difference from 'lodash/difference';
-import metricsgraphics from 'metrics-graphics';
```

Comment:
The `metricsgraphics.css` import can be removed from `ui/entry-perf.js` now (since it's imported in `Graph.jsx` already)